cst438-project-airline/src/main/java/cst438/controller/AirlineRestController.java

```diff
@@ -78,11 +80,25 @@ public Reservation makeReservation(@RequestParam("flightId") int flightId,
       @RequestParam("passengerFirstName") String passengerFirstName,
       @RequestParam("passengerLastName") String passengerLastName) {
 
+    // Check if this seat is available to book
+    if (!airlineService.isSeatAvailable(seatId)) {
```

Comment:
`Read this after the comment in the test file`

oh, ok i see why you stubbed this in the test.

Reply:
Oh got it. This comment was written after the previous one?